has-many.mdx

```diff
@@ -101,12 +101,66 @@ relationship from our example, add a `User` to the `Transfer` class:
 
 ### Many-to-many
 
-Many-to-many relations are defined by has-many relationship on both sides of the relation.
-As before, you don't need to specify the join condition a second time if the join
-condition is symmetric.
+The recommended way to define a many-to-many relationship is through a joining feature class.
+For instance, to define a many-many relationship between `Actors` and `Movies`, you
+could write the following feature classes:
 
-<PyDiffEditor html={manyToManyDiff} />
+```python
+from chalk.features import features, DataFrame
+
+@features
+class Actor:
+  id: str
+  appearances: "DataFrame[ActorInMovie]"
+  full_name: str
+
+  # this will be used to demonstrate one of the ways the joining feature can be populated
+  movie_ids: list[int]
+
+@features
+class Movie:
+  id: str
+  title: str
+
+@features
+class ActorInMovie:
+  id: str
+  actor_id: Actor.id
+  movie_id: Movie.id
+  movie: Movie
+```
 
 <TipInfo>
-  Here you need to use quotes around `DataFrame[Author]` to use a forward reference.
+  Here you need to use quotes around `DataFrame[ActorInMovie]` to use a forward reference.
 </TipInfo>
+
+This joining feature class can either by populated by:
+
+A SQL Resolver
+```sql
+-- resolves: ActorInMovie
+-- source: PG
+SELECT a.id||'_'||m.id as id, a.id as actor_id, m.id as movie_id FROM actors a CROSS JOIN movies m;
```

Comment:
not every actor was in every movie, so it's a little weird to me to use a cross-join in this example. could we make this example select from a table like "movie_roles" or "characters"?